addons/hr_expense/views/hr_expense_views.xml

```diff
@@ -69,12 +66,12 @@
                     <field name="unit_amount_display" string="Unit Price" optional="hide" widget="monetary" options="{'currency_field': 'company_currency_id'}"/>
                     <field name="quantity" optional="hide"/>
                     <field name="tax_ids" optional="hide" widget="many2many_tags" groups="account.group_account_invoice,account.group_account_readonly"/>
-                    <field name="amount_tax_company" optional="hide" groups="account.group_account_invoice,account.group_account_readonly"/>
+                    <field name="amount_tax_company" sum="Total Taxes"  optional="hide" groups="account.group_account_invoice,account.group_account_readonly"/>
                     <field name="attachment_number" nolabel="1" class='fa fa-paperclip pe-0' attrs="{'invisible': [('attachment_number', '=', 0)]}"/>
-                    <field name="total_amount_company" widget='monetary' options="{'currency_field': 'company_currency_id'}" decoration-bf="1"/>
-                    <field name="total_amount" widget='monetary' options="{'currency_field': 'currency_id'}" optional="hide" decoration-bf="1" groups="base.group_multi_currency"/>
+                    <field name="total_amount_company" sum="Total Amount" widget='monetary' options="{'currency_field': 'company_currency_id'}" decoration-bf="1"/>
+                    <field name="total_amount" sum="Total Amount in Currency" widget='monetary' options="{'currency_field': 'currency_id'}" optional="hide" decoration-bf="1" groups="base.group_multi_currency"/>
                     <field name="currency_id" optional="hide" groups="base.group_multi_currency"/>
-                    <field name="state" optional="show" readonly="1" decoration-info="state == 'draft'" decoration-success="state in ['reported', 'approved', 'done']" decoration-danger="state in 'refused'" widget="badge"/>
+                    <field name="state" optional="show" readonly="1" decoration-info="state in ['draft', 'reported']" decoration-success="state in ['approved', 'done']" decoration-warning="state == 'submitted'" decoration-danger="state == 'refused'" widget="badge"/>
```

Comment:
Indent? Or feel free to keep this shitty indentation by consistency with the rest of the file